story/version.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-version = '0.9.0'
+version = '0.9.1'
```

Comment:
```suggestion
version = '0.10.0'
```

This won't be a patch release ;-)